frontend/src/screens/EventDetails.tsx

```diff
@@ -1,21 +1,151 @@
-            <ThemedText>{`Mock Event ${eventNumber}`}</ThemedText>
-        </View>
+        <MainContainer>
+            <HeaderContainer>
+            <AntDesign
+                        style={{ marginTop: "15%", marginLeft: "3%" }}
+                        name="caretleft"
+                        size={24}
+                        color="white"
+                        onPress={() => {navigation.navigate("Login")}}
+                    />
+            <Entypo 
+                        style={{ marginTop: "15%", marginRight:"5%" }}
+                        name="heart" 
+                        size={28} 
+                        color={isLiked ? "red": "black"}
+                        onPress={()=>{
+                            setIsLiked(!isLiked);
+                        }}
+                        />
+            </HeaderContainer>
+            <Animated.ScrollView showsVerticalScrollIndicator={false} ref={scrollRef} style={{height:"100%", backgroundColor:"white"}} scrollEventThrottle={16}>
+                    
+                    <Animated.Image
+                        style={[{height:250, width: "100%", backgroundColor:"red"},imageAnimatedStyle]}
+                        source={require("../../assets/Campus_Buddy_Logo.png")}
+                    />
+
+                    <View style={{height: "10%", width: "100%", backgroundColor:"white", flexDirection:'row', justifyContent:"space-between" }}>
+                        <EDetails>
+                            <Text style={{fontFamily:"Roboto-Medium", fontSize:16, marginBottom:5}}>{eventData.title}</Text>
+                            <Text style={{fontFamily:"Roboto-Medium", fontSize:16, marginBottom:5}}>{eventData.date}</Text>
+                            <TagContainer>
+                                <Feather name="map-pin" size={12} color="black" style={{marginRight:5}}/>
+                                <Text style={{fontFamily:"Roboto-Medium", fontSize:10}}>{eventData.location}</Text>
+                            </TagContainer>
+                        </EDetails>
+                        <EClubDetails>
+                            <Image 
+                                style={{height: 30, width:30, backgroundColor:"red", borderRadius:90,marginBottom:5}}
+                                source={require("../../assets/Campus_Buddy_Logo.png")}
+                            />
+                            <Text style={{fontFamily:"Roboto-Medium", fontSize:18}}>{eventData.clubName}</Text>
+                        </EClubDetails>
+                    </View>
+                    <View style={{borderTopColor:"#B0CFFF", borderTopWidth:1,flexDirection:"row", height:50,backgroundColor:"white", alignItems:"center"}}>
+                        <Ionicons name="people-outline" size={30} color="black" style={{marginLeft:10}}/>
+                        <Text style={{fontFamily:"Roboto-Medium", fontSize:16, marginLeft:5}}>Attendance: 200</Text>
```

Comment:
This attendance counter needs to be render dynamically using the eventData object